### ISSUE TITLE
Text color always white using Material Dialog Library

### ISSUE THREAD

**Issue by jordyamc** [afollestad/aesthetic#59]
- [x] I have verified there are no duplicate active or recent bugs, questions, or requests
- [x] I have verified that I am using the latest version of Aesthetic (see the README)
- [x] I have given my issue description title
- [x] I have read over the documentation (before asking questions on how to do something).

###### Include the following:
 - Aesthetic version: `0.4.7`
 - Device OS version: `8.0.0`
 - Device Manufacturer: `Emulator`
 - Device Name: `Android TV`
```java
if (Aesthetic.isFirstTime())
            Aesthetic.get()
                    .isDark(theme.isDark)
                    .colorWindowBackgroundRes(android.R.color.transparent)
                    .textColorPrimaryRes(theme.isDark ? android.R.color.primary_text_dark : android.R.color.primary_text_light)
                    .textColorPrimaryInverseRes(theme.isDark ? android.R.color.primary_text_light : android.R.color.primary_text_dark)
                    .textColorSecondaryRes(theme.isDark ? android.R.color.secondary_text_dark : android.R.color.secondary_text_light)
                    .textColorSecondaryInverseRes(theme.isDark ? android.R.color.secondary_text_light : android.R.color.secondary_text_dark)
                    .colorAccent(theme.accent).apply();
```

```xml
<style name="Theme.AppCompat.Translucent" parent="Theme.AppCompat.NoActionBar">
        <item name="android:background">@android:color/transparent</item>
        <item name="android:windowNoTitle">true</item>
        <item name="android:windowBackground">@android:color/transparent</item>
        <item name="android:colorBackgroundCacheHint">@null</item>
        <item name="android:windowIsTranslucent">true</item>
    </style>
```
 
###### Reproduction Steps
Use Material Dialog library
Set Translucent style on activity

###### Expected Result
The Dialog takes the text color as the Aesthetic method

###### Actual Result
The content and list text only takes white color

**Comment by jordyamc** [comment]
![screenshot_1502761309](https://user-images.githubusercontent.com/6831873/29298969-2c70af5e-8131-11e7-87bf-859f3e8c5f66.png)

Also the dialog is taking the grong accent color

**Comment by boswelja** [comment]
What happens if you flip these colors around?
`theme.isDark ? android.R.color.primary_text_dark : android.R.color.primary_text_light`
I mention that as I always get confused as to which text color is which

**Comment by jordyamc** [comment]
nothing, i add it after some try and error but still shows the text in white

**Comment by boswelja** [comment]
Probably related to Android TV, that emphasises a dark UI, right? Why not use a dark dialog theme?

**Comment by jordyamc** [comment]
well it can be a temporal fix, but i was thinking to add a Dark/Ligth theme function to my app

**Comment by jordyamc** [comment]
Also the ProgressBar and RadioButton of thw dialogs are not using the AccentColor

And it seems the methods .widgetColor() and .choiceWidgetColor() are not changing the colors

**Comment by ch4ndu** [comment]
This happens in a PreferenceFragment as well...CheckBoxes are always white while unchecked.

**Comment by boswelja** [comment]
Try specifying text colors manually, I think Android TV overrides both text colors @jordyamc

**Comment by ch4ndu** [comment]
I saw this is on a Nexus 6p running Nougat...
While using MaterialDialog library, all checkbox's'(single and multi selection) unchecked state is white on white. Also, EditText has white on white text.

Apart from this, I have a PreferenceFragment in my app and the CompoundButton's there behave the same way with Aesthetic enabled on the activity hosting the Fragment. Disabling Aesthetic is the workaround for me currently. 

I've set only colorPrimary, colorAccent, statusbarColor and navbarColor using Aesthetic btw. Toggling isDark() didn't here.

            Aesthetic.get()
                    .colorPrimaryRes(R.color.primary)
                    .colorAccentRes(R.color.accent)
                    .colorNavigationBarAuto()
                    .colorStatusBarAuto()
                    .isDark(false)
                    .apply();

Is there a way to let Aesthetic modify only the colors/parameters we manually set instead of changing all default colors? Example - I want Aesthetic to modify only colorPrimary, colorAccent in my app.
